src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,4 @@
+package duke.exception;
+
+public class DukeException extends Exception {
```

Comment:
since this is a custom exception, you need to provide Javadoc comments to indicate why it is there in the codebase